modified_cdc_protocol/operation_types/setup_pcr_plate/documentation.rb

```diff
@@ -1 +1,16 @@
-Documentation here. Start with a paragraph, not a heading or title, as in most views, the title will be supplied by the view.
\ No newline at end of file
+  def default_operation_params
```

Comment:
Why is this in `documentation.rb`?

Reply:
This may be an error.  But I had put it there so when making plans we could see our "Options" of what we could change and what the defaults were.   I think it could be better written than this but this was quick and easy for now.